step_configure_hw.go

```diff
@@ -36,7 +40,16 @@ func (s *StepConfigureHW) Run(state multistep.StateBag) multistep.StepAction {
 		confSpec.MemoryMB = int64(ram)
 	}
 
-	state.Put("confSpec", confSpec)
+	task, err := vm.Reconfigure(ctx, confSpec)
```

Comment:
Done. But if no options were specified, I print the log message anyway, but just another one.